### DISCUSSION TITLE
Visualizing Cherry Blossom Bloom Dates with {ggplot2} and {plotnine}

### DISCUSSION THREAD

**Discussion by christophscheuch** [has2k1/plotnine#826] [2024 Plotnine Contest]
### Authors

Christoph Scheuch

### Links

https://www.tidy-intelligence.com/

### Full description

While I was traveling Japan earlier this year, I had the pleasure to experience sakura, the famous Japanese cherry blossoms. These blossoms are highly significant in Japanese culture, symbolizing the transient nature of life due to their brief blooming period. Sakura typically bloom in spring, from late March to early May, depending on the region. The blossoming of cherry trees is celebrated with hanami, a traditional custom of enjoying the beauty of flowers, often involving picnics under the blooming trees.

Climate change is causing sakura to bloom earlier and for shorter periods due to rising temperatures and shifting phenological patterns. This disrupts traditional hanami festivals, affects pollination success, and leads to regional variations in bloom timing, with urban areas often experiencing more pronounced shifts.

In this blog post, I visualize data with respect to cherry blossom blooming periods over regions and time. I provide a comprehensive guide on loading and preparing the corresponding data for analysis using both R and Python. The analysis spans from 1953 to 2023, leveraging the tidyverse family of packages for R and the ibis-framework for Python, complemented by plotnine for visualizations.

https://blog.tidy-intelligence.com/posts/sakura-visualizations/

### Code repository

https://github.com/tidy-intelligence/blog/tree/main/posts/sakura-visualizations

**Comment by has2k1** [comment]
![unnamed-chunk-12-1](https://github.com/user-attachments/assets/bf4b29b4-14bc-48f0-aedb-369ba922068e)
<hr/>

![unnamed-chunk-14-1](https://github.com/user-attachments/assets/53161ce7-78fe-487e-8d8f-e443d3cb18ac)

**Comment by EconomiCurtis** [comment]
Ahoy @christophscheuch , 
I am one of the contest organizers. 
If you'd like, we'd be happy to send you a little Plotnine and Posit care package of hex-shaped stickers. Just a small thanks for taking part. https://forms.gle/M8QmM286MSHbkDRd7

**Comment by has2k1** [comment]
@christophscheuch, The winning submission has been announced [here](https://posit.co/blog/winner-of-the-2024-plotnine-plotting-contest/). Thank you for taking part in the contest.
